src/media/js/apps_buttons.js

```diff
@@ -8,6 +8,7 @@ define('apps_buttons',
     var console = log('buttons');
     var gettext = l10n.gettext;
     var apps_model = models('app');
+    var desktopLoginPopup = false;
```

Comment:
nit: it would be cool to call this ``webLoginPopup`` since it really means 'when on platforms without mozPay' which could include mobile webkit. If that's confusing then no bother.